### DISCUSSION TITLE
Connecting domain using ANAME/ALIAS method (AWS Route 53)

### DISCUSSION THREAD

**Discussion by patjakubik** [vercel/vercel#4276] [General] [answered]
My nameservers need to stay as they are, so the only method left is using ANAME/ALIAS.

AWS Route 53 doesn't support ANAME/ALIAS, so what the other options are?

Anyone had such problem?

**Comment by coetry** [comment, marked as answer]
Hi @patryk-smc I shared an alternative way of getting your domains connected to your projects in [#4243](https://github.com/zeit/now/discussions/4243).

**Reply by patjakubik** [reply]
Thanks. For future readers: I shared some more details how I solved that problem in #4243 too.
